agent/api/container/container.go

```diff
@@ -96,6 +96,8 @@ type HealthStatus struct {
 type Container struct {
 	// Name is the name of the container specified in the task definition
 	Name string
+	// RuntimeID is the docker id of the container
+	RuntimeID string
```

Comment:
this probably needs [statefile version](https://github.com/aws/amazon-ecs-agent/blob/fa463fee4c79352636ad2e3851144a9ac0804be2/agent/statemanager/state_manager.go#L91) bump and test added [here](https://github.com/aws/amazon-ecs-agent/blob/dev/agent/statemanager/state_manager_test.go).